Widgets/widget-RunningPlaybooks.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "running-playbooks",
+  "version": -1,
+  "name": "Running playbooks",
+  "dataType": "incidents",
+  "widgetType": "trend",
```

Comment:
not sure if trend or number. running playbooks in last 7 days trend is not helpfull